packages/html/src/HTMLTooltip.ts

```diff
@@ -244,6 +246,22 @@ export class HTMLTooltip extends HTMLWidget {
         return bbox;
     }
 
+    private _closing = false;
+    mouseout() {
+        this._closing = true;
+        this._tooltipElement.on("mouseover", () => {
+            this._closing = false;
+        });
+        this._tooltipElement.on("mouseout", () => {
+            this.mouseout();
+        });
+        setTimeout(()=>{
+            if(this._closing){
+                this.visible(false);
+            }
+        }, this.closeDelay());
+    }
```

Comment:
What is this fixing?

Reply:
When the cursor leaves the vertex to (potentially) interact with the content of the tooltip `this.closeDelay()` is the number of milliseconds it has to re-enter the vertex or enter the tooltip. (before the vertex disappears)